### DISCUSSION TITLE
Resume Bundle X ATS score

### DISCUSSION THREAD

**Discussion by kavyapurushottama** [recodehive/resume-pitch#103] [Q&A] [answered]
Resume- [Resume.pdf](https://github.com/user-attachments/files/17084493/Resume.pdf)
I am a final year student applying for various job opportunities. Please review my resume and suggest some changes in it if required.
LinkedIn- [LinkedIn](https://www.linkedin.com/in/pkavya-774217247)

**Comment by sanjay-kv** [comment, marked as answer]
I didnt see any topmate request for this 
could you check that again: 
if you wanna book again: https://topmate.io/sanjaykv/1047284/pay
use the same freecode you used before.
